documentation/decisions/adr/2023-09-07-data-replication-tool.md

```diff
@@ -26,15 +26,24 @@ The Beta.Grants.Gov platform will need to consume data and apply additional load
 
 ## Options Considered
 
+### Data Replication
+
 - Use Production Database
 - Use AWS DMS (Database Migration Service)
 - Create new data pipelines from data sources
+- Import/Export DB snapshots weekly
+
+### Data Traffic
 
-## Decision Outcome
+- AWS VPC Pairing
+- AWS PrivateLink
+- Network Gateway with VPN
+
+## Decision Outcome - Data Replication
 
 Chosen option: Use AWS DMS, because it is the only option that allows us to deliver within our period of performance and doesn't impact the production database's ability to perform its existing role.
```

Comment:
I think we should move this towards "recommendation" or "proposed" right now until we receive sign-off from MH.

Reply:
I thought our [ADR Process](https://github.com/HHS/grants-equity/tree/main/documentation/decisions#adr-process) is to write them as if they are approved outcomes. And the decision is not a decision until it's merged. That's why we abandoned using various statuses in favor of always drafting with the status "Active" (or optionally "Superseded By…" later). Otherwise we need to reframe the perspective of ADRs when/after they're merged.

Reply:
I would like to incorporate MH into our process as much as possible on this, but I also want to be sensitive to the friction that some of our practices may inadvertently create. I think a solution here is to change it to `Draft` during this review period, include MH in the review process before merging, and when everyone is aligned, change it to `Active` when merging. In my mind that threads the needle of making this process accessible to them but also following our established practice. 
I'm also open to MH changing our minds on the right approach here.

Reply:
I'm fine with that, let's just make it clear when discussing that this is a proposal for discussion.